packages/fluentui/react-northstar/test/specs/commonTests/isConformant.tsx

```diff
@@ -408,8 +427,13 @@ export default function isConformant(
           [listenerName]: handlerSpy,
           [EVENT_TARGET_ATTRIBUTE]: true,
         };
-
-        const component = mount(<Component {...props} />);
+        const ComponentHandled = passesUnhandledPropsTo;
+        const component = passesUnhandledPropsTo
+          ? mount(<ComponentHandled {...props} />)
+          : mount(<Component {...props} />);
+        const componentPropTypes = passesUnhandledPropsTo
+          ? { ...Component.propTypes, ...passesUnhandledPropsTo.propTypes }
```

Comment:
I would like to avoid there usage of `passesUnhandledPropsTo.propTypes`, is there any reason to have it them?